src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,165 @@
+
+### Bridges
+
+One particular area of Layer-2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2.
+
+### Sharding
+
+Sharding Ethereums blockchain has long been part of the development roadmap. However, new scaling solutions such as "Danksharding" are currently taking center stage.
+
+### Hardware
+
+Running nodes on modest hardware is fundamental to keeping Ethereum decentralized. Therefore, active research into minimizing the hardware requirements to run nodes is an important area of research.
+
+### Links to current research
+
+[Ethereum on ARM](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/)
+
+[Ethresear.ch Scaling](https://ethresear.ch/c/sharding/6)
+
+[Ethresear.ch Layer 2](https://ethresear.ch/c/layer-2/32)
+
+[Ethereum Sharding Research Compendium](https://notes.ethereum.org/@serenity/H1PGqDhpm?type=view)
+
+[Danksharding (Polynya)](https://polynya.medium.com/danksharding-36dc0c8067fe)
+
+## Security
+
+Security is a broad topic that might include spam/scam prevention, wallet security, hardware security, crypto-economic security, bug hunting and testing of applications and client software and key-management. Contributing to knowledge in these areas will help stimulate mainstream adoption.
+
+### Cryptography & zkp
+
+Zero-knowledge proofs and crytopgraphy are critical for building privacy and security into Ethereum and its applications. Zero-knowledge is a relatively young but fast-moving space with many open research and development opportunities.
+
+### Wallets
+
+User-wallets can be browser extensions, desktop and mobile apps or smart contracts on Ethereum. There is active research into social recovery wallets that reduce some of the risk associated with individual-user key management.
+
+### Links to current research
+
+[Three Attacks on PoS Ethereum](https://arxiv.org/abs/2110.10086)
+
+[ethresear.ch Security](https://ethresear.ch/tag/security)
+
+[Ethresear.ch ZK](https://ethresear.ch/c/zk-s-nt-arks/13)
+
+[zkp.science](https://zkp.science/)
+
+[Zero Knowledge podcast](https://zeroknowledge.fm/)
+
+## Community, Education and Outreach
+
+Onboarding new users onto Ethereum requires new educational resources and approaches to outreach. This might include blog posts and articles, books, podcasts, memes, teaching resources events and anything else that builds communities, welcomes new starters and educates people about Ethereum.
+
+### UX/UI
+
+For Ethereum to onboard more people the UX/UI must be improved. This requires designers and UI and product experts to re-examine the design of wallets and apps.
+
+### Links to current research
+
+[Ethresear.ch UX/UI](https://ethresear.ch/c/ui-ux/24)
+[Ethereum.org community page](https://ethereum.org/en/learn/)
+
+## Economics
```

Comment:
This site has a good list of additional potential subtopics within Economics: https://ethereum.github.io/rig/